libs/wlgen/wlgen/rta.py

```diff
@@ -43,11 +43,20 @@ class Phase(_Phase):
     pass
 
 class RTA(Workload):
+    """
+    Class for creating RT-App workloads
+    """
 
     def __init__(self,
                  target,
                  name,
                  calibration=None):
+        """
+        :param target: Devlib target to run workload on.
+        :param name: Human-readable name for the workload
```

Comment:
If you want to remove them all around, I'm all it for that better me too ;-)
However, as I said... don't spent too much time on that ;-)